src/modules/storefront/components/Products.tsx

```diff
@@ -0,0 +1,202 @@
+import { CSSProperties, useEffect } from 'react';
+
+import { Autoplay, Navigation } from 'swiper';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+import { Product, ProductsData, ProductsDefault } from '../interfaces';
+
+import 'swiper/css';
+import 'swiper/css/navigation';
+
+import './Products.css';
+
+const getCardInfo = (
+  data: ProductsData['card'],
+  defaultData: ProductsDefault['card']
+): ProductsDefault['card'] => {
+  return {
+    name: data?.name !== undefined ? data.name : defaultData.name,
+    description:
+      data?.description !== undefined
+        ? data.description
+        : defaultData.description,
+    category:
+      data?.category !== undefined ? data.category : defaultData.category,
```

Comment:
Não precisa validar se é diferente de undefined. na teoria se for null ou undefined ele atribui o que esta depois do ??